dags/dts_maximo_reporting.py

```diff
@@ -149,6 +149,18 @@
         mount_tmp_dir=False,
     )
 
+    t7 = DockerOperator(
+        task_id="maximo_locations_to_socrata",
+        image=docker_image,
+        docker_conn_id="docker_default",
+        auto_remove="force",
+        command=f"python etl/maximo_to_socrata.py --query locations",
+        environment=env_vars,
+        tty=True,
+        force_pull=False,
```

Comment:
was going to ask about this, but I see that the first task pulls and subsequent tasks don't need to 👍